src/cmd/services/m3aggregator/main/main.go

```diff
@@ -83,19 +86,46 @@ func main() {
 		SetMetricsSamplingRate(cfg.Metrics.SampleRate()).
 		SetReportInterval(cfg.Metrics.ReportInterval())
 
-	// Create the raw TCP server options.
-	rawTCPAddr := cfg.RawTCP.ListenAddress
-	rawTCPServerScope := scope.SubScope("rawtcp-server").Tagged(map[string]string{"server": "rawtcp"})
-	iOpts := instrumentOpts.SetMetricsScope(rawTCPServerScope)
-	rawTCPServerOpts := cfg.RawTCP.NewServerOptions(iOpts)
+	var (
+		m3msgAddr        string
+		m3msgServerOpts  m3msg.Options
```

Comment:
Maybe combine these and put address into options?

Reply:
Might punt on this for now, good suggestion though.